css/style.css

```diff
@@ -1,22 +1,34 @@
 /* global */
 @import url("https://fonts.googleapis.com/css?family=Roboto");
 
+html {
+  height: 100%;
+}
+
 body {
-  /*
```

Comment:
this commented section was mainly to see how the layout of the site looked like without the margins.